sharding/p2p/feed.go

```diff
@@ -0,0 +1,33 @@
+package p2p
+
+import (
+	"reflect"
+
+	"github.com/ethereum/go-ethereum/event"
+)
+
+// P2P feed is a one to many subscription feed of the argument type.
+//
+// Messages received via p2p protocol are sent to subscribers by these event
+// feeds. Message consumers should not use event feeds to reply to or broadcast
+// messages. The p2p server will not relay them to peers. Rather, use the
+// Send() or Broadcast() method on p2p.Server.
```

Comment:
Maybe elaborate a little bit more on why this is the case? I know that it is because the sending of information needs to use the peer info? Also elaborate on the difference between send and broadcast here for those that are unclear.

Reply:
I don't think this is the right place for that. If the reader is interested in those methods, they can read them from the same godoc (they just have to scroll).

The problem I have here is that if we change the details of those methods then we will need to update this comment too. 

I realize that these methods are not well documented, but that is because they haven't been written yet.